leptos/src/hydration/mod.rs

```diff
@@ -144,6 +149,7 @@ pub fn HydrationScripts(
             r#type="application/wasm"
             crossorigin=nonce.clone().unwrap_or_default()
         />
+        <link rel="stylesheet" id="leptos" href=format!("{root}/{pkg_path}/{css_file_name}.css") />
```

Comment:
People may or may not want their CSS to apply to all pages, I don't think we can add this here by default